jenkins-scripts/dsl/gz-collections.yaml

```diff
@@ -96,7 +96,7 @@ collections:
       configs:
         - focal
         - brew
-        - win
```

Comment:
IIUC this PR this line has to be kept(? Then you would have focal, brew, win and win_citadel configs

Reply:
Note that this `config` section [is defining the configurations for Citadel](https://github.com/gazebo-tooling/release-tools/pull/1097/files#diff-f69908edf9a24c4bb5dca1219f6d179a15eb2e65739e410c65df6d343d3d2b63L29-L96) so win is replaced by win_citadel.

You are right about that now we have 4 configurations available, with `win`still [defined generally](https://github.com/gazebo-tooling/release-tools/pull/1097/files#diff-f69908edf9a24c4bb5dca1219f6d179a15eb2e65739e410c65df6d343d3d2b63L526) in the `ci_configs` section and used for Fortress, Garden, Harmonic, Ionic and __upcoming__.

Reply:
Oh, I see, thanks for the explanation